yarn.lock

```diff
@@ -6067,7 +6067,7 @@ find-root@1.1.0:
   resolved "https://registry.yarnpkg.com/find-root/-/find-root-1.1.0.tgz#abcfc8ba76f708c42a97b3d685b7e9450bfb9ce4"
   integrity sha512-NKfW6bec6GfKc0SGx1e07QZY9PE99u0Bft/0rzSD5k3sO/vwkVUpDUKVm5Gpp5Ue3YfShPFTX2070tDs5kB9Ng==
 
-find-up@^2.0.0, find-up@^2.1.0:
```

Comment:
Unless you're modifying `package.json` or bumping a transitive dependency, you shouldn't have changes in the lockfile. Please make sure you're running `yarn install --frozen-lockfile` when installing and only include the lockfile in your PR if you're deliberately modifying packages as part of your work.